src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -610,7 +621,11 @@ def test_resource_group_deployment(self):
             'params_invalid': os.path.join(curr_dir, 'simple_deploy_parameters_invalid.json').replace('\\', '\\\\'),
             'dn': self.create_random_name('azure-cli-resource-group-deployment', 60),
             'dn2': self.create_random_name('azure-cli-resource-group-deployment', 60),
-            'Japanese-characters-tf': os.path.join(curr_dir, 'Japanese-characters-template.json').replace('\\', '\\\\')
+            'dn3': self.create_random_name('azure-cli-resource-group-deployment', 60),
+            'Japanese-characters-tf': os.path.join(curr_dir, 'Japanese-characters-template.json').replace('\\', '\\\\'),
+            'ts': '/subscriptions/a1bfa635-f2bf-42f1-86b5-848c674fc321/resourceGroups/TemplateSpecsCLI/providers/Microsoft.Resources/templateSpecs/BasicTemplateSpecs/versions/1.1.0',
+            'context': 'a1bfa635-f2bf-42f1-86b5-848c674fc321',
```

Comment:
Could those resources be created dynamically instead of a fixed value?

Reply:
The resource cannot yet be dynamically created. We need to add the template-spec command module (CRUD operations) before the test cases can be updated.